CHANGELOG.md

```diff
@@ -1,26 +1,33 @@
-# 1.51.0 (2019-08-28)
```

Comment:
Hmm. Apparently the v1.51.0 changelog entry was removed 🤔

Reply:
Indeed, great you've spot that. Fixing

Reply:
Updated